docs/source/using/troubleshooting.adoc

```diff
@@ -194,3 +194,29 @@ flag in the `minishift start` command. Alternatively, the problem might be
 an outdated version of VirtualBox.
 
 To avoid this issue, it is recommended to use VirtualBox 5.1.12 or later.
+
+[[insufficient-privileges]]
+== Hyper-V: Administrator privileges insufficient to run Hyper-V
+
```

Comment:
I think there are some pieces missing here. In its core the problem is that even if you have Admin privileges, you need to run the shell in elevated mode to start Minishift. This is what @thesteve0 expected. You open your normal power shell and just type 'minshift start'. However, this does not work. In fact whatever the error is in this case, is what should be added to this trouble shooting entry.

So once the user hits the error, he has to choices. Either open a new shell in elevated mode or he does what we describe below and add himself to the HyperV admin group. If so, he can then open a user shell (non elevated) and start Minishift.

So "Hyper-V: Administrator privileges insufficient to run Hyper-V", is actually standalone misleading. As admin you can for sure start Hyper-V. The issue is trying to start Minishift in a normal user mode shell. I'd start with checking what the error message is. This is probably the title for the trouble shooting entry. Then I describe the two choices one has to fix it. Run shell in elevated mode or add myself to the Hyper-V admin group.

@Preeticp Does this make sense

Right now the title is confusing and there are logical gaps in why you can/should add yourself to the Hyper-V admin group.